e2e/whisper/whisper_jail_test.go

```diff
@@ -48,46 +41,33 @@ type WhisperJailTestSuite struct {
 func (s *WhisperJailTestSuite) StartTestBackend(opts ...e2e.TestNodeOption) {
 	s.BackendTestSuite.StartTestBackend(opts...)
 
+	s.Timeout = time.Minute * 5
```

Comment:
Is there a need to store this in the struct given that it's only used in a single place?

Reply:
Yes and no. Before I removed a separate test for test case 4 it was used at least two times :D It can be refactored later if it turns out to not be used.